modules/videojsVideoProvider.js

```diff
@@ -146,10 +146,7 @@ export function VideojsProvider(providerConfig, vjs_, adState_, timeState_, call
     // ~ Sort of resolved check if the player has a source to tell if the placement is instream
     // Still cannot reliably check what type of placement the player is if its outstream
     // i.e. we can't tell if its interstitial, in article, etc.
-    if (player.src()) {
-      video.placement = PLACEMENT.INSTREAM;
```

Comment:
can we set `video.plcmt` to 2 ? If `player.src()` evaluates to `truethy` then we know there is content.

Reply:
resolved on line below